sktime/classification/distance_based/_time_series_neighbors.py

```diff
@@ -88,6 +63,28 @@ class KNeighborsTimeSeriesClassifier(_KNeighborsClassifier, BaseClassifier):
         - [callable] : a user-defined function which accepts an
           array of distances, and returns an array of the same shape
           containing the weights.
+    algorithm : str, optional. default = 'brute'
+        search method for neighbours
+        one of {'auto’, 'ball_tree', 'kd_tree', 'brute'}
+    distance : str or callable, optional. default ='dtw'
+        distance measure between time series
+        if str, must be one of the following strings:
+            'euclidean', 'squared', 'dtw', 'ddtw', 'wdtw', 'wddtw',
+            'lcss', 'edr', 'erp', 'msm'
+        this will substitute a hard-coded distance metric from sktime.distances
+        When mpdist is used, the subsequence length (parameter m) must be set
+            Example: knn_mpdist = KNeighborsTimeSeriesClassifier(
+                                metric='mpdist', metric_params={'m':30})
```

Comment:
- `mpdist` is not in the docstring or the `DISTANCES_SUPPORTED` variable. I do not know what resolution is.
- I think `metric` should be `distance`. (and more metric_params)

Reply:
I think this was copied from sklearn, fixed